moto/redshift/models.py

```diff
@@ -941,6 +943,36 @@ def delete_tags(self, resource_name, tag_keys):
         resource = self._get_resource_from_arn(resource_name)
         resource.delete_tags(tag_keys)
 
+    @staticmethod
+    def _generate_random_alphanum():
+        printable = list(string.ascii_letters + string.digits + string.punctuation)
```

Comment:
Minor nitpick - but the method name (alphanum) is incorrect if punctuation is allowed

Reply:
True, my bad 😅